fearless/Modules/Staking/Analytics/Common/AnalyticsPeriodView/AnalyticsPeriodView.swift

```diff
@@ -0,0 +1,151 @@
+import UIKit
+
+protocol AnalyticsPeriodViewDelegate: AnyObject {
+    func didSelect(period: AnalyticsPeriod)
+}
+
+final class AnalyticsPeriodView: UIView {
+    weak var delegate: AnalyticsPeriodViewDelegate?
+
+    private var periods: [AnalyticsPeriod] = []
+
+    let collectionView: UICollectionView = {
```

Comment:
Not sure that we need CollectionView with cell caching overhead. We might use just scroll view with rounded buttons if we really afraid of that it might be cases when period doesn't fit page width